DIPs/DIP1xxy-mvf.md

```diff
@@ -0,0 +1,322 @@
+                              // After stage 2: Error
+                              // Remedy: `boolValue = cast(bool)1L`
+
+    boolValue = 0UL;          // Existing implementation: OK
+                              // After stage 1: Deprecation warning
+                              // After stage 2: Error
+                              // Remedy: `boolValue = cast(bool)0UL`
+
+    boolValue = 1UL;          // Existing implementation: OK
+                              // After stage 1: Deprecation warning
+                              // After stage 2: Error
+                              // Remedy: `boolValue = cast(bool)1UL`
+
+    boolValues = [0,1,0,1];   // Existing implementation: OK
+                              // After stage 1: Deprecation warning
+                              // After stage 2: Error
+                              // Remedy: `boolValue = cast(bool[])[0,1,0,1]`
+
+    bitArray = BitArray([0,1,0,1]); // Existing implementation: OK
+                                    // After stage 1: Deprecation warning
+                                    // After stage 2: Error
+                                    // Remedy: `bitArray = BitArray(cast(bool[])[0,1,0,1])`.
+                                    // `int[]` constructor and function overloads can also be added
+                                    // to `BitArray`'s implementation to avoid requiring the user to
+                                    // insert an explicit cast.
+
+    f(E.a);                   // Existing implementation: `calls f(bool)`
+                              // After stage 1: Deprecation warning, still `calls f(bool)`
+                              // After stage 2: calls `f(int)`
+                              // Remedy to keep existing behavior: `f(cast(bool)E.a)`
+
+    f(E.b);                   // Existing implementation: `calls f(bool)`
+                              // After stage 1: Deprecation warning, still `calls f(bool)`
+                              // After stage 2: calls `f(int)`
+                              // Remedy to keep existing behavior: `f(cast(bool)E.a)`
+
+    g(a - b);                 // Existing implementation: `calls g(bool)`
+                              // After stage 1: Deprecation warning, still `calls g(bool)`
+                              // After stage 2: calls `g(long)`
+                              // Remedy to keep existing behavior: `f(cast(bool)(a - b))`
+}
+
+enum YesNo : bool { no, yes } // Existing implementation: OK
+                              // After stage 1: Deprecation warning
+                              // After stage 2: Error
+                              // Remedy: `enum YesNo : bool { no = false, yes = true }`
+
+```
+Stage 1's implementation will require updating the [Deprecated Features](https://dlang.org/deprecate.html) page, an update to the [language specification](https://dlang.org/spec/type.html#bool), and en entry in the changelog.
```

Comment:
"and en entry" => and an entry

Some suggested text for the web page and the spec would be beneficial.

Reply:
> Some suggested text for the web page and the spec would be beneficial.

The existing deprecated features, and the recent changelogs set a pretty good precedent for what is expected.  I'd also prefer to defer that wordsmithing until I hear how this DIP is received by both the community and the language maintainers.